middleware/http/server.go

```diff
@@ -135,10 +144,10 @@ func (h handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		code := ri.getStatusCode()
 		sCode := strconv.Itoa(code)
 		if code > 399 {
-			zipkin.TagError.Set(sp, sCode)
+			h.errHandler(sp, nil, code)
```

Comment:
If using this errHandler, why not have a default errHandler take care of code inspection (default code > 399 == error) so you can swap out for your own implementation...

Reply:
Mixed feelings here, I am concerned about calling `errHandler` when code is 200 (for example) and the only way out I think of is that we have two separated handlers, one for errors and other for status code. Since this is HTTP specific, I think it makes sense to keep it for actual http "error" codes. Any other thoughts?

Reply:
keep as is then